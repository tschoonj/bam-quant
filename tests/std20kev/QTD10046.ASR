File: C:\xrfdaten\2014KW30\chris\std20kev\std10046.spe
Start: 21.07.2014 18:22:48
Stop: 21.07.2014 18:24:28
BESSY: 298.024005100116 298.9759553
IONI: 2.934222E-11 2.943707E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97994
Run: 100000
Dead: 2
ICR: 770
OCR: 0
Accumulated BESSY: 59646.6836255
Accumulated Ioni: 5.86561057E-09
N Accumulation: 200
Normfactor:  3.4795036e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pd
exp_x 111.260398864746 111.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  46   2      2.838     21122.       151.        .78
