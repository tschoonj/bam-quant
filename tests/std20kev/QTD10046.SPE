$DATA:
0 2000
22988
14788
6335
1795
356
40
4
0
0
0
0
0
0
0
0
0
4
24
26
22
35
29
31
29
33
28
26
16
27
20
19
16
14
21
23
16
23
16
26
19
19
19
18
25
19
24
21
12
25
23
25
29
60
65
83
65
52
46
32
46
46
42
44
39
41
42
33
32
22
27
28
21
16
15
29
23
19
20
13
12
22
26
20
18
15
15
18
19
18
15
11
10
12
18
22
16
19
13
18
21
21
21
17
21
22
31
38
35
32
24
18
25
17
19
11
31
25
29
30
45
62
75
92
112
121
114
96
95
82
68
87
105
95
122
201
307
658
1236
1954
2776
3387
3382
3083
2340
1817
1484
1590
1805
2088
2195
1993
1769
1532
1155
933
614
467
365
322
207
169
91
87
73
83
82
62
45
22
32
20
29
28
45
61
61
63
75
58
40
24
19
15
4
7
3
3
4
5
6
6
11
4
7
3
9
4
5
5
8
6
11
6
6
10
12
2
6
7
10
7
6
9
5
7
8
7
7
12
13
10
7
11
6
8
8
6
9
9
9
10
10
8
9
11
12
8
10
6
11
12
8
5
9
11
11
8
5
8
12
18
10
14
10
10
11
10
7
11
11
12
11
9
11
6
10
13
15
10
11
10
11
8
10
14
9
18
8
10
11
13
7
9
12
18
9
3
6
7
13
17
8
17
12
15
10
6
12
5
11
9
12
13
18
8
19
13
14
8
8
18
20
16
20
29
44
40
39
40
34
26
22
22
17
11
18
13
13
13
11
11
12
12
16
9
15
16
16
10
9
21
18
14
14
17
9
9
9
19
14
20
19
12
18
10
9
10
20
10
12
6
12
12
16
21
13
8
18
17
20
20
16
17
16
20
11
16
14
16
13
11
13
10
16
17
11
15
20
24
12
19
15
9
15
18
27
17
25
18
22
25
14
16
32
14
21
10
20
16
15
24
21
15
25
20
15
15
26
21
23
21
23
24
23
21
20
19
18
11
16
21
10
18
13
18
17
21
16
17
20
24
28
21
14
10
13
22
21
12
16
14
18
12
16
17
21
12
19
26
20
30
42
44
52
69
87
74
97
87
83
48
57
61
47
38
57
69
92
109
116
148
108
112
116
86
70
50
41
40
17
28
20
24
15
27
17
26
11
22
13
23
14
13
17
27
24
15
19
11
22
19
15
17
16
18
19
23
22
22
27
15
22
23
18
27
20
21
24
24
15
22
22
24
25
24
24
23
20
27
31
27
29
41
49
61
66
64
73
90
82
68
70
65
57
58
63
61
48
57
43
69
62
66
62
82
80
71
96
91
94
74
74
78
54
50
49
45
44
31
29
35
24
15
27
20
26
23
24
16
25
22
19
23
20
16
20
26
20
15
23
23
23
27
23
20
20
24
12
21
17
14
23
23
12
17
23
22
24
19
13
24
26
23
23
31
23
18
23
20
24
28
10
18
22
27
29
35
27
28
24
29
31
28
32
30
30
20
27
25
26
21
20
24
24
17
18
20
27
37
34
33
31
24
33
44
31
34
26
29
13
25
39
22
21
16
31
26
22
30
31
25
24
31
21
26
21
32
25
24
21
19
20
25
23
24
31
18
24
13
23
16
20
18
12
30
23
25
29
20
21
23
29
21
28
14
24
27
25
28
22
16
21
29
23
23
18
26
21
23
26
14
32
25
30
22
29
13
29
34
22
23
24
23
22
35
19
22
23
26
24
30
24
24
31
30
30
28
41
27
25
31
33
33
21
23
37
37
42
44
49
65
72
110
114
117
132
178
222
238
247
227
232
208
172
178
146
99
78
67
66
44
42
45
31
30
27
39
32
31
37
31
30
35
40
33
38
37
31
51
33
40
33
32
25
26
31
33
39
35
30
38
24
34
32
28
31
34
28
28
26
35
20
23
18
11
13
14
19
14
11
5
16
12
4
9
11
8
7
7
3
10
13
10
6
11
5
10
9
8
9
12
8
12
12
9
16
21
15
20
26
32
37
18
42
36
58
43
31
35
34
21
23
18
21
21
17
20
25
12
22
13
13
16
11
11
13
12
10
17
18
8
13
12
15
16
13
11
13
8
17
18
14
20
18
13
20
19
24
31
20
29
29
28
26
24
28
21
21
25
32
22
38
31
25
32
28
32
38
32
42
46
32
34
43
47
31
56
47
45
48
42
44
40
56
52
52
52
58
60
56
53
59
59
49
52
63
59
68
58
65
74
72
92
86
102
105
119
139
180
224
261
295
292
378
406
460
522
470
497
509
547
522
513
530
452
403
437
390
343
296
266
206
174
160
131
98
84
72
53
31
27
23
18
9
7
7
6
4
3
2
2
2
0
0
1
1
0
0
2
1
3
1
2
4
2
3
3
4
7
8
10
11
11
9
13
16
12
13
13
9
10
5
8
7
2
1
3
2
1
1
2
1
0
1
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
2
2
2
0
4
1
6
3
1
1
2
0
3
0
0
1
0
1
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10046.spe
Start: 21.07.2014 18:22:48
Stop: 21.07.2014 18:24:28
BESSY: 298.024005100116 298.9759553
IONI: 2.934222E-11 2.943707E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 97994
Run: 100000
Dead: 2
ICR: 770
OCR: 0
Accumulated BESSY: 59646.6836255
Accumulated Ioni: 5.86561057E-09
N Accumulation: 200
Normfactor:  3.4795036e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Pd
 
exp_x 111.260398864746 111.2604
exp_y 92.3237991333008 92.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  46   2      2.838     21122.       151.        .78
