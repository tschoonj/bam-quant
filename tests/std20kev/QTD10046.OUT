     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:48  บ   
     บ  Spectrum: QTD10046.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        3 iterations done    บ   
     บ  ChiSquare =    16.8   last change =   .02%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ     -113.8 ๑   100.0   บ    -113.8 ๑     6.2   บ   
     บ  GAIN (eV/ch)  บ     20.848 ๑   2.000   บ    20.848 ๑    .045   บ   
     บ det NOISE (eV) บ       87.7 ๑    40.0   บ      87.7 ๑    17.1   บ   
     บ  FANO factor   บ       .121 ๑    .050   บ      .121 ๑    .034   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Pd-L1                              4238. ๑     97.           บ   
     บ    L1M3        3.073    .51040        2163. ๑     51.     72.2  บ   
     บ              152.861    124.93         122.           4.88E-01  บ   
     บ    L1M2        3.045    .30011        1272. ๑     31.     71.7  บ   
     บ              151.518    124.64         123.           4.79E-01  บ   
     บ    L1N3        3.553    .18949         803. ๑     21.     53.9  บ   
     บ              175.886    129.78         116.           6.12E-01  บ   
     บ                                                                 บ   
     บ  2 Pd-L2                             10929. ๑    139.           บ   
     บ    L2M4        2.990    .87676        9582. ๑    123.     53.2  บ   
     บ              148.880    124.07         124.           4.60E-01  บ   
     บ    L2N4        3.329    .09802        1071. ๑     17.     91.8  บ   
     บ              165.141    127.54         119.           5.57E-01  บ   
     บ    L2M1        2.660    .01951         213. ๑     11.     17.3  บ   
     บ              133.051    120.60         106.           3.42E-01  บ   
     บ    L2N1        3.244    .00572          62. ๑     11.     75.6  บ   
     บ              161.064    126.68         120.           5.32E-01  บ   
     บ                                                                 บ   
     บ  3 Pd-L3                             24306. ๑    174.           บ   
     บ    L3M5        2.838    .86901       21122. ๑    152.       .8  บ   
     บ              141.589    122.49         105.           4.05E-01  บ   
     บ    L3N5        3.172    .10154        2468. ๑     21.     78.4  บ   
     บ              157.610    125.95         121.           5.16E-01  บ   
     บ    L3M1        2.503    .02201         535. ๑     11.     12.2  บ   
     บ              125.520    118.91         108.           2.79E-01  บ   
     บ    L3N1        3.087    .00744         181. ๑     11.     74.4  บ   
     บ              153.533    125.07         122.           4.93E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                               0. ๑      9.           บ   
     บ    L2N4        5.848   1.00000           0. ๑     13.      2.3  บ   
     บ              285.989    150.83         104.           8.94E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
